### PR TITLE
Rest framework configuration and basic recipe model

### PR DESCRIPTION
**Pull request by Tichau**

### CODE REVIEW

File: backend/apps/project/views.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+
+class UserViewSet(viewsets.ModelViewSet):
+    """
+    API endpoint that allows users to be viewed or edited.
+    """
+    queryset = User.objects.all().order_by('-date_joined')
+    serializer_class = UserSerializer
+
+
+class GroupViewSet(viewsets.ModelViewSet):
```
Comment (nadege) [inline backend/apps/project/views.py]: This two views are a security issue, they are not protected by a permission so anybody can edit users and groups :fearful:

File: backend/apps/recipes/views.py [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+from apps.recipes.models import Recipe
+from .serializers import RecipeSerializer
 
-# Create your views here.
+
+class RecipeViewSet(viewsets.ReadOnlyModelViewSet):
+    """
+    API endpoint that allows recipes to be viewed or edited.
+    """
+    queryset = Recipe.objects.all().order_by('-title')
```
Comment (nadege) [inline backend/apps/recipes/views.py]: why not just 'title' ?

File: docs/backend.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 
 To create a superuser, run `django-admin createsuperuser`.
 To access to the admin part, go to http://localhost:8000/admin
+If you want to modify the admin password, run `django-admin changepassword admin`
```
Comment (nadege) [inline docs/backend.md]: Are you cloning the django documentation ? :)

**Comment by nadege** [comment]
don't forget to run pylint !

File: docs/backend.md (hunk 3 above)
Comment (Tichau) [inline docs/backend.md]: ok I remove this file :)

File: backend/apps/recipes/views.py (hunk 2 above)
Comment (Tichau) [inline backend/apps/recipes/views.py]: you're right!

File: backend/apps/project/views.py (hunk 1 above)
Comment (Tichau) [inline backend/apps/project/views.py]: For now, all the API (even recipes) is locked for authenticated users only (so the admin user).
I wanted to work on permissions later, are you ok with that? or did I start this work now?

**Comment by Tichau** [comment]
I've done the fixes and wait for you response about permissions :)

**Pull request merged**